lib/table.js

```diff
@@ -3,6 +3,8 @@
 var wrap = require('word-wrap');
 
 var Table = require('cli-table2'),
+    argv  = require('yargs').argv,
```

Comment:
Put the definition of the sortBy option into cli.js (with documentation). I suggest a choice of 'name' and 'score', with a default of 'name'. Then make sortProperty an argument to getNewTable.